manifests/init.pp

```diff
@@ -159,6 +161,7 @@
   Variant[Enum['unmanaged'], Stdlib::Absolutepath] $defaultzonepath = $dns::params::defaultzonepath,
   Optional[Enum['only', 'first']] $forward                          = undef,
   Array[Dns::Forwarder] $forwarders                                 = [],
+  Optional[Variant[String, Boolean]] $listen_on                     = undef,
```

Comment:
Why a boolean instead of just an optional string?
```suggestion
  Optional[String] $listen_on                                       = undef,
```

Is it for consistency with `$listen_on_v6`? I'm not sure we need to care about that too much.

Reply:
It is for consistency with listen_on_v6 but I agree we can probably ignore it. Let me update the change.